test/upload/signature_test.rb

```diff
@@ -2,7 +2,7 @@
 
  describe 'upload-signature' do
    let(:connection) do
-     Excon.new("#{CONFIG.UPLOAD_HOST}/prod/signature")
+     Excon.new("#{CONFIG.UPLOAD_HOST}/signature")
```

Comment:
How was CD passing, if this was broken?

Reply:
Unclear. Probably the change to the new upload host never got deployed